src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -36,7 +41,9 @@ public QuestionDTO getQuestion(@PathVariable("id") Long id) {
     @ApiOperation(value = "Add a new question")
     @PostMapping("/")
     public QuestionDTO addQuestion(@RequestBody QuestionDTO questionDTO) {
-        Question question = questionService.createQuestion(questionConverter.convertToEntity(questionDTO));
+        Question question = questionService
+                .createQuestion(questionConverter.convertToEntity(questionDTO), questionDTO.getAnswers());
+        answerRepository.updateCorrect(questionDTO.getCorrect());
```

Comment:
Can we find a way to mark a response as a correct without duplicating the data.

BTW: some questions doesn't have correct answers

Reply:
Do you mean that some questions have no answers at all?